contents/handbook/people/spending-money.md

```diff
@@ -145,6 +145,14 @@ You can spend up to \$200/month to work in cafés or coworking spaces if working
 
 If you live in the US, a particularly good way to use this budget is to cover the cost of Amex Platinum which provides WeWork access. Outside of the US, you may sign your own agreement or buy day vouchers as needed. We will not cover costs beyond this amount of money.
 
+## Meetups
+
+As a global, all-remote team, we don't get many regular opportunities to socialise with each other outside of our bi-annual off-sites.
+
+We encourage team members to try and meet up with each other when practical, and provide everyone with an $250 monthly budget to cover the cost of travel and meals. Beyond staying within the spending limit, please just spend the money in good faith and like it is your own. Any unused budget does not roll over each month. 
```

Comment:
There's a good example on No rules, rules on why "like it is your own" is not a good way to think about money (in a nutshell, we all spend money in vastly different ways), but something along the lines of "what's best for the company" or in line with our overall spending money spirit should be great